index.html

```diff
@@ -274,6 +274,229 @@ <h3>
+          <var>processedProtocolHandlers</var>.
+          </li>
+          <li>[=list/For each=] <var>protocol_handler</var> (<a>protocol
+          handler description</a>):
+            <ol>
+              <li>If |protocol_handler|["protocol"] or
+              <var>protocol_handler</var>["url"] is undefined,
+              [=iteration/continue=].
+              </li>
+              <li>Let (<dfn>normalizedProtocol</dfn>, <dfn>normalizedUrl</dfn>)
+              be the result of running [=normalize protocol handler
+              parameters=] with <var>protocol_handler</var>["protocol"], <var>
+                protocol_handler</var>["url"] using <var>manifest URL</var> as
+                the base URL, and [=this=]'s relevant [=environment settings
+                object=]. If the result is failure, [=iteration/continue=].
+              </li>
+              <li>If [=normalizedUrl=] is not [=manifest/within scope=] of
+              <var>start URL</var>, [=iteration/continue=].
+              </li>
+              <li>If [=normalizedUrl=] already exists in
+              <var>processedProtocolHandlers</var>, [=iteration/continue=].
+              </li>
+              <li>[=List/Append=] |protocol_handler| to
+              <var>processedProtocolHandlers</var>.
+              </li>
+            </ol>
+          </li>
+          <li>[=list/For each=] |processedProtocolHandlers|, the user agent
+          SHOULD [=register a protocol handler=].
+          </li>
+        </ol>
+        <p>
+          A user agent SHOULD ask users for permission before registering a
+          <a>protocol handler description</a> <var>protocol_handler</var>s as
+          the default handler for a protocol with the host operating system. A
+          user agent MAY truncate the list of <a>protocol handler
+          description</a> <var>protocol_handlers</var> presented in order to
+          remain consistent with the conventions or limitations of the host
+          operating system.
+        </p>
+        <aside class="example">
+          <p>
+            In the following example, the developer has included two
+            <a>protocol handler description</a> <var>protocol_handler</var>s.
+            Assuming the the manifest's URL is
+            <samp>https://example.com/manifest.webmanifest</samp>:
+          </p>
+          <ul>
+            <li>The first protocol handler would register to handle "web+music"
+            URLs (e.g.: web+music://#1234). When activated, the user agent
+            would instantiate a new <a>top-level browsing context</a> and
+            navigate to
+            <samp>https://example.com/play?songId=web+music://%231234</samp>.
+            </li>
+            <li>The second protocol handler would be ignored, as the protocol
+            provided does not start with "web+" and is not part of the
+            [=safelisted schemes=].
+            </li>
+          </ul>
+          <pre class="example json">
```

Comment:
I think remove "example" here as it looks weird in the rendered document to have two example headers - just "json" should be good.